lib/ReactViews/Workbench/Controls/OpacitySection.jsx

```diff
@@ -35,6 +36,8 @@ const OpacitySection = observer(
       const { t } = this.props;
       if (
         !hasTraits(item, RasterLayerTraits, "opacity") ||
+        // This is needed - as there is some issue with `LoadableStratum` adding `RasterLayerTraits` randomly
```

Comment:
Are you running a tutorial for me @nf-s ? Because I think this is caused by not sticking documentation you posted about this morning: 
https://github.com/TerriaJS/terriajs/blob/49e238f25a9be5ef0c48b1a92746ceac14315c86/doc/contributing/model-layer.md#traits
& https://github.com/TerriaJS/terriajs/blob/3b2568e9ff46a20ad0050429f146432b5d346dd9/lib/Traits/TraitsClasses/RasterLayerTraits.ts#L39

Reply:
haha yep, sorry forgot about this change

Reply:
Removed the condition and comment.
All of the trait inheritance is sorted in this PR https://github.com/TerriaJS/terriajs/pull/5711